wmi_check/README.md

```diff
@@ -9,6 +9,9 @@ Get metrics from your Windows applications/servers with Windows Management Instr
 * Visualize their performance.
 * Correlate their activity with the rest of your applications.
 
+**Note:** It is recommended that the [PDH check][14] be used instead in all cases due to its significantly less overhead
```

Comment:
There is a line break at the end of this line that should be removed

Reply:
Done!